### PR TITLE
Report more information from the server than "unknown error"

### PR DESCRIPTION
**Pull request by exjimsk**
If more information is available (i.e.: can be unmarshaled and asserted as a string – there may be cases where serverReply.Value.Message is a not a string), it should be returned by webDriver.execute()

### CODE REVIEW

**Comment by exjimsk** [comment]
fixed the type of serverReply.Value to interface{} and assert type when an error with more details can be reported.

File: remote.go [hunk 1]
```diff
@@ -192,11 +194,21 @@ func (wd *remoteWD) execute(method, url string, data []byte) ([]byte, error) {
 		if err != nil {
 			return nil, errors.New(fmt.Sprintf("Bad server reply status: %s", response.Status))
 		}
+
 		message, ok := remoteErrors[reply.Status]
 		if !ok {
 			message = fmt.Sprintf("unknown error - %d", reply.Status)
 		}
 
+		if val, ok := reply.Value.(map[string]interface{}); ok {
```
Comment (tebeka) [inline remote.go]: Can you move this to a utility function and use it?

```go
func extractMessage(value) string {
    m, ok := value.(map[string]interface{})
    if !ok {
      return ""
    }
    msg, _ := m["message"].(string)
    return msg // I can't convert we'll get the empty string
}
```
...
```
msg := extractMessage(reply.Value)
if len(msg) > 0 {
    message = msg
}
```

**Comment by tebeka** [comment]
Thanks! I have a small comment, can you please fix?

**Comment by exjimsk** [comment]
fixed

**Pull request merged**